runtime/runtime/src/config.rs

```diff
@@ -132,7 +152,10 @@ pub fn total_send_fees(
                 }
             },
             DeleteKey(_) => cfg.delete_key_cost.send_fee(sender_is_receiver),
-            DeleteAccount(_) => cfg.delete_account_cost.send_fee(sender_is_receiver),
+            DeleteAccount(_) => {
+                // TODO add cfg[feature]
+                cfg.delete_account_cost.send_fee(sender_is_receiver) + cfg.transfer_cost.send_fee(sender_is_receiver)
```

Comment:
You should use `send_transfer_fee(cfg, sender_is_receiver, receiver_id, current_protocol_version)` here

Reply:
Discussed offline